.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,18 @@
 <!-- Common title prefixes/annotations:
 
-  WIP:
-  bugfix:
-  🏗️ maintenance:
+WIP: work in progress
 
-  (⚠️ devops)  = changes in devops config required before deploying
+Consider prefix your PR message with an emoticon
+  🐛 bugfix
+  ✨ new feature
+  🔨 refactoring
+  🏗️ maintenance
+  📚 documentation
+
```

Comment:
What should be used if something needs to be changed in the GitHub Actions? Cult you put a comment as well for that action?

Reply:
These are just initial **suggestions**. Below those entries this is a link to a github repo with a more comprehensive list ...  Feel free to add more here.